service/Microsoft.Azure.Devices/test/Microsoft.Azure.Devices.Api.Test/ClientWebSocketTransportTests.cs

```diff
@@ -116,6 +116,7 @@ public void ClientWebSocketTransportWriteWithoutConnectTest()
         [TestMethod]
         [TestCategory("CIT")]
         [TestCategory("WebSocket")]
+        [Ignore]
```

Comment:
Please open a github issue to track for investigation into these disabled test cases.